src/cmd/cli/command/commands.go

```diff
@@ -350,11 +350,20 @@ var RootCmd = &cobra.Command{
 		// Use "defer" to track any errors that occur during the command
 		defer func() {
 			var errString = ""
+			logProps := []track.Property{}
 			if err != nil {
 				errString = err.Error()
+
+				// on error, also log the recent terminal messages
+				messages := term.DefaultTerm.GetAllMessages()
+				logProps = append(logProps, track.ChunkMessages("logs", messages)...)
```

Comment:
I understand there is some limit on the number of messages in a single property, but I don't understand the limit.

I assume it's a simple limit on max string length per property. If so, then this implementation can fail to avoid that limit (one log line can be very very long). I wonder if we should concatenate all of the logs together and take the tail up to that limit instead.

Is the limit logic more complex than that? If so, what are the details?